CustomerJourney/.gitignore

```diff
@@ -4,4 +4,4 @@ yarn-error.log
 **/yarn-error.log
 yarn.lock
 .npmrc
-.env
\ No newline at end of file
+.env**
```

Comment:
add an empty line after line 7

Reply:
Why? Other .gitignore files does not have a end-of-file empty line

Reply:
I made it a habit to try to remember to add a line at the end of .gitignore. You don't have to.